canon/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "canonical"
 version = "0.6.0"
 authors = ["Kristoffer Ström <kristoffer@dusk.network>"]
 edition = "2018"
-readme = "README.md"
+readme = "../README.md"
```

Comment:
Since we're here maybe we also want to add to the README / lib the shields, e.g. the one about the CI, the repo, and the docs. See:

https://raw.githubusercontent.com/dusk-network/dusk-abi/master/README.md

and:

https://raw.githubusercontent.com/dusk-network/dusk-abi/master/src/lib.rs

Depends who is the source of truth (the lib's comment or the readme) you can just include it once.